packages/merge-styles/src/Stylesheet.ts

```diff
@@ -158,28 +159,28 @@ export class Stylesheet {
   public insertRule(
     rule: string
   ): void {
-    const element = this._getElement();
-    const injectionMode = element ? this._config.injectionMode : InjectionMode.none;
-
-    switch (injectionMode) {
-      case InjectionMode.insertNode:
-        const { sheet } = element!;
-
-        try {
-          // tslint:disable-next-line:no-any
-          (sheet as any).insertRule(rule, (sheet as any).cssRules.length);
-        } catch (e) {
-          /* no-op on errors */
-        }
-        break;
-
-      case InjectionMode.appendChild:
-        _createStyleElement(rule);
-        break;
-
-      default:
-        this._rules.push(rule);
-        break;
+    const { injectionMode } = this._config;
+    const element = injectionMode !== InjectionMode.none ? this._getStyleElement() : undefined;
+
+    if (element) {
+      switch (this._config.injectionMode) {
+        case InjectionMode.insertNode:
+          const { sheet } = element!;
+
+          try {
+            // tslint:disable-next-line:no-any
+            (sheet as any).insertRule(rule, (sheet as any).cssRules.length);
```

Comment:
`sheet!.insertRule(rule, sheet!.cssRules.length)` instead of all the casts to any please